pv_site_api/main.py

```diff
@@ -462,6 +464,15 @@ def get_pv_forecast(
     if len(forecasts) == 0:
         return Response(status_code=204)
 
+    # Adjust forecast values if normalize_by_capacity is True
+    if normalize_by_capacity:
+        site = get_site_by_uuid(session, site_uuid)  # Replace with actual function to get site by UUID
```

Comment:
is this right? or do you need help with this?

Reply:
> is this right? or do you need help with this?

It's not complete yet, I haven't written the function to get the site by id yet

Reply:
> this might help. - https://github.com/openclimatefix/pv-site-datamodel/blob/main/pvsite_datamodel/read/site.py#L14

Thank you, ill check this out !